testing/src/cypress.config.ts

```diff
@@ -34,5 +30,8 @@ export default defineConfig({
     token_url:
       "https://dev.oidc.gov.bc.ca/auth/realms/jxoe2o46/protocol/openid-connect/token",
     update_institution_url: "/api/institutions/institution",
+    studentURL: "https://test-aest-sims.apps.silver.devops.gov.bc.ca/student/login",
+    username: "",
```

Comment:
These variables seem to be too generic, are they supposed to be used for student login?
If this is not the focus of the PR we can handle this later.

Reply:
Yes they are for login. we just need to set the specific  set test users for automation. But we can handle this into our env like one above